relay-java/src/main/java/com/genymobile/gnirehtet/Main.java

```diff
@@ -50,7 +51,12 @@ String getDescription() {
             @Override
             void execute(CommandLineArguments args) throws Exception {
                 Log.i(TAG, "Installing gnirehtet client...");
-                execAdb(args.getSerial(), "install", "-r", "gnirehtet.apk");
+                File apk = new File("/opt/gnirehtet/gnirehtet.apk");
```

Comment:
If we install the _gnirehtet_ package, then locally download another version, it will unexpectedly use `gnirehtet.apk` from `/opt`. Therefore, IMO, it should check in the local directory first.

Also, it should not check this path on other platforms like Windows.

Maybe the APK path should be provided by an environment variable instead. What do you think?

In would require some wrapper script like:

    GNIREHTET_APK=/usr/lib/gnirehtet/gnirehtet.apk /usr/lib/gnirehtet/gnirehtet "$@"

If the variable is not set, it would just use the local directory. This would make it configurable for packaging according to different rules for different distributions.

I don't know what's the best way to do it.